homeassistant/components/ccm15/climate.py

```diff
@@ -0,0 +1,166 @@
+"""Climate device for CCM15 coordinator."""
+import logging
+from typing import Any
+
+from ccm15 import CCM15DeviceState
+
+from homeassistant.components.climate import (
+    FAN_AUTO,
+    FAN_HIGH,
+    FAN_LOW,
+    FAN_MEDIUM,
+    SWING_OFF,
+    SWING_ON,
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import CONST_CMD_FAN_MAP, CONST_CMD_STATE_MAP, DOMAIN
+from .coordinator import CCM15Coordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class CCM15Climate(CoordinatorEntity[CCM15Coordinator], ClimateEntity):
+    """Climate device for CCM15 coordinator."""
+
+    def __init__(
+        self, ac_host: str, ac_index: int, coordinator: CCM15Coordinator
+    ) -> None:
+        """Create a climate device managed from a coordinator."""
+        super().__init__(coordinator)
+        self._ac_host: str = ac_host
+        self._ac_index: int = ac_index
+        self._attr_should_poll = True
+        self._attr_temperature_unit = UnitOfTemperature.CELSIUS
+        self._attr_has_entity_name = True
+        self._attr_name = f"{self._ac_index}"
```

Comment:
Can you give an example of such name? Did you look into DeviceInfo? Also


```suggestion
        self._attr_name = str(ac_index)
```